web/src/components/common/CommitProgress.tsx

```diff
@@ -42,8 +43,16 @@ const CommitProgressContainer = styled.div`
   }};
 
   text-align: right;
-  font-size: ${({largerFont}: CommitProgressContainerProps) =>
-    largerFont ? '1em' : '0.8em'};
+  font-size: ${({fontSize}: CommitProgressContainerProps) => {
+    switch (fontSize) {
+      case 'large':
+        return '2em';
+      case 'medium':
+        return '1em';
+      default:
```

Comment:
nit: Maybe we can do this so that it's clearer that 'small' goes under the default case too
```suggestion
      case 'small':
      default:
```

Reply:
@hzxcaryn Okay!